agent.py

```diff
@@ -1319,13 +1300,15 @@ def updateMeanIncome(self, sugarIncome, spiceIncome):
         self.spiceMeanIncome = (alpha * spiceIncome) + ((1 - alpha) * self.spiceMeanIncome)
 
     def updateNeighbors(self):
-        self.updateVonNeumannNeighbors()
-        self.updateMooreNeighbors()
+        self.socialNeighbors = []
+        for neighborCell in self.cell.neighbors.values():
```

Comment:
Is there a clean way to do this as a list comprehension one-liner?

Reply:
Sure!